Apps/FCM/ios/FCMSampleApp/MyAppPushNotificationsHandler.swift

```diff
@@ -10,10 +10,6 @@ public class MyAppPushNotificationsHandler : NSObject {
 
   @objc(setupCustomerIOClickHandling)
   public func setupCustomerIOClickHandling() {
-    // This line of code is required in order for the Customer.io SDK to handle push notification click events.
-    // We are working on removing this requirement in a future release.
-    // Remember to modify the siteId and apiKey with your own values.
-    CustomerIO.initialize(withConfig: SDKConfigBuilder(cdpApiKey: Env.cdpApiKey).build())
     MessagingPushFCM.initialize(withConfig: MessagingPushConfigBuilder().build())
```

Comment:
I think we should still leave some comments for `MessagingPushFCM.initialize` to explain why it is needed.

Reply:
That makes sense. Added a comment